### PR TITLE
PTS integrity check crashing with nil pts fix

### PR DESCRIPTION
**Pull request by bartkrak**
solves: https://github.com/membraneframework/membrane_core/issues/791

### CODE REVIEW

**Review by FelonEkonom — CHANGES_REQUESTED** [review]
Changes are OK. Add test checking, if encoder can handle `nil` timestamps.

**Review by FelonEkonom — APPROVED** [review]

File: test/membrane_element_fdk_aac/pts_test.exs [hunk 1]
```diff
@@ -0,0 +1,77 @@
+defmodule Membrane.FLAC.Parser.IntegrationTest do
+  use ExUnit.Case
```
Comment (varsill) [inline test/membrane_element_fdk_aac/pts_test.exs:2]: How about?
```suggestion
  use ExUnit.Case, async: true
```

**Review by varsill — APPROVED** [review]
